### PR TITLE
added prompts for update products and complete order to ui.js

### PR DESCRIPTION
**Pull request by cait-rocco**
I made the following changes:

updated the menu and added prompts for update product and complete order to ui.js.

To test:

currently not testable until functionality exist to activate a user. when available, try each menu option and see that it prompts you for the correct info. no functionality in yet to add the user entered items to the database.

### CODE REVIEW

File: app/ui.js [hunk 1]
```diff
@@ -314,4 +295,99 @@ let productPopPrompt = () => {
       resolve(results);
     })
   });
+};
+
+module.exports.displayOrder = (total) => {
+  console.log(`
+  ${magenta('**  Your order total is,' total, 'Ready to purchase (Y/N)  **')}`);
```
Comment (mozecode) [inline app/ui.js]: Hey, take a look at this line-- comma after is needs to be moved outside of the quotes.

File: app/ui.js [hunk 2]
```diff
@@ -314,4 +295,99 @@ let productPopPrompt = () => {
...
+    .then( (activeCustomer) => {
+      console.log('this customer is now active:', activeCustomer)
+      //run active customer function that opens the customerMenuHandler
+    });
+  } else if (userInput.choice == '3') {
+    prompt.stop();
+  }
+};
+
+let completeOrderPrompt = () => {
```
Comment (mozecode) [inline app/ui.js:278]: Change the name of this and where it's called-- I keep getting this error: Identifier 'completeOrderPrompt' has already been declared

File: app/ui.js [hunk 3]
```diff
@@ -238,61 +219,161 @@ let newProductPrompt = () => {
...
+      required: true
+    }], function(err, results) {
+      if (err) return reject(err);
+      resolve(results);
+    })
+  });
+};
+
+module.exports.displayOrder = (total) => {
+  console.log('Your order total is,' total, 'Ready to purchase (Y/N)  **');
```
Comment (Rian501) [inline app/ui.js]: This comma needs to be outside the quote marks plz

**Review by Rian501 — CHANGES_REQUESTED** [review]
With that one comma change, it seems to be working insofar as I am able to test it!

**Pull request merged**